packages/core/src/MenuSectionHeader/MenuSectionHeader.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import cx from 'classnames'
+import { colors, spacers } from '@dhis2/ui-constants'
+
+import { Divider } from '../Divider/Divider.js'
+
+/**
+ * @module
+ * @param {MenuSectionHeader.PropTypes}
+ * @returns {React.Component}
+ *
+ * @example import { MenuSectionHeader } from '@dhis2/ui
+ *
+ * @see Specification: {@link https://github.com/dhis2/design-system/blob/master/molecules/menu.md|Design system}
+ * @see Live demo: {@link /demo/?path=/story/components-core-menusectionheader--default|Storybook}
+ */
+const MenuSectionHeader = ({ className, dataTest, dense, label }) => (
+    <li data-test={dataTest} className={cx(className, { dense })}>
+        <Divider dense={dense} />
```

Comment:
Should this one also have the `Divider` when it's the first section? I think it'd make sense to wrap this one with a div that's `display: none` when the `li` is `:first-child`

Reply:
Yeah it seems like the first shouldn't have that. I would prefer not rendering over display none if possible. It seems like the poor mans version of not rendering.

Reply:
I've made this conditional via a `hideDivider` prop in b358c47. Lemme know if that's OK.

Reply:
I think I'm fine with that. Just one thought that we could discuss:

As probably all menu sections that are the first child shouldn't show the divider, we'd force the developers to always provide a prop to the first component, which seems like an odd dev practice. Maybe there are better solutions for this. For example the component that will eventually contain the section (I think that's the `Menu` component that already clones the children?) could also pass a `hideDivider: true` to the first child (at `index === 0`) if the component does not have an explicit value for that prop (`typeof child.props.hideDivider === 'undefined'`)

Reply:
I like that 👍 
Added in df49ce8. I'm only setting `hideDivider` to `true` under the following condition:
```javascript
child.props.hideDivider !== 'boolean' && index === 0
```
So if for whatever reason the developer needs a divider above the first header, he can explicitly set `hideDivider={false}`